### PR TITLE
Added healthcheck endpoint

### PR DESCRIPTION
**Pull request by davidgouge**

### CODE REVIEW

**Comment by EwanNoble** [comment]
Missed a setting: https://github.com/SkillsFundingAgency/das-apply-service/blob/master/azure/template.json#L118-L121

File: azure/sandbox-template.json [hunk 1]
```diff
@@ -283,6 +283,10 @@
                             {
                                 "name": "UseSandboxServices",
                                 "value": "yes"
+                            },
+                            {
+                                "name": "WEBSITE_SWAP_WARMUP_PING_PATH",
+                                "value": "/health"
                             }
```
Comment (jack-education) [inline azure/sandbox-template.json]:
```suggestion
                            }
                            {
                                "name": "WEBSITE_SWAP_WARMUP_PING_STATUSES",
                                "value": "200"
                            }
```

File: azure/template.json [hunk 2]
```diff
@@ -103,6 +103,10 @@
                             {
                                 "name": "EnvironmentName",
                                 "value": "[toUpper(parameters('environmentName'))]"
+                            },
+                            {
+                                "name": "WEBSITE_SWAP_WARMUP_PING_PATH",
+                                "value": "/health"
                             }
```
Comment (jack-education) [inline azure/template.json]:
```suggestion
                            },
                            {
                                "name": "WEBSITE_SWAP_WARMUP_PING_STATUSES",
                                "value": "200"
                            }
```

File: azure/template.json [hunk 3]
```diff
@@ -157,6 +161,10 @@
...
```
Comment (jack-education) [inline azure/template.json]:
```suggestion
                            },
                            {
                                "name": "WEBSITE_SWAP_WARMUP_PING_STATUSES",
                                "value": "200"
                            }
```

**Pull request merged**